### PR TITLE
HBW-224 Stop polling on window blur

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

**Comment by mxsmirnoff** [comment]
I have video/gif proof.
It's too big for attaching, but I can show you.

File: hbw/app/javascript/packs/hbw/connection.js [hunk 1]
```diff
@@ -233,8 +233,20 @@ modulejs.define('HBWConnection', ['jQuery'], (jQuery) => {
...
       this.serverURL = this.options.host + this.options.path;
       this.channels = {};
+
+      document.addEventListener('visibilitychange', () => {
+        if (document.visibilityState === 'visible') {
+          this.startPolling();
+        } else {
+          this.stopPolling();
+        }
+      });
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/connection.js]: Move this to separate method please.

File: hbw/app/javascript/packs/hbw/connection.js [hunk 2]
```diff
@@ -233,8 +233,20 @@ modulejs.define('HBWConnection', ['jQuery'], (jQuery) => {
...
     }
 
+    stopPolling = () => Object.keys(this.channels).forEach(c => this.channels[c].stop());
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/connection.js]: `Object.values` I suppose.

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ Removed `tasks/form` enrtypoint. Entrypoint `tasks` returns tasks with form.
 
 ### Features
 - [#390](https://github.com/latera/homs/pull/390) Add spinner on tasks loading.
+- [#392](https://github.com/latera/homs/pull/392) Stop polling on inactive tab.
```
Comment (TimAle) [inline CHANGELOG.md]: This version has already been released.

**Pull request merged**